src/main/java/com/untamedears/realisticbiomes/utils/RealisticBiomesGUI.java

```diff
@@ -96,6 +97,31 @@ public void showRBOverview() {
 				}
 			}
 			lore.add(ChatColor.DARK_AQUA + "Biome Multiplier: " + ChatColor.GRAY + biomeMultiplier);
+			if (plant.getAllowBoneMeal() || !plant.getNeedsLight()
+					|| plant.getSoilBoniPerLevel().size() != 0 || plant.getMaximumSoilLayers() != 0
+					|| plant.getMaximumSoilBonus() != 0 || plant.getGreenHouseRates().size() != 0) {
+				lore.add(ChatColor.DARK_GREEN + "---");
+			}
+			if (plant.getAllowBoneMeal()) {
+				lore.add(ChatColor.DARK_AQUA + "Allow Bone Meal: " + ChatColor.GRAY + "true");
+			}
+			if (!plant.getNeedsLight()) {
+				lore.add(ChatColor.DARK_AQUA + "Needs Light: " + ChatColor.GRAY + "false");
+			}
+			for (Map.Entry<Material, Double> entry : plant.getSoilBoniPerLevel().entrySet()) {
+				lore.add(ChatColor.DARK_AQUA + "Soil Bonus: " + ChatColor.GRAY
+						+ ItemNames.getItemName(entry.getKey()) + " (" + Double.toString(entry.getValue()) + ")");
+			}
+			if (plant.getMaximumSoilLayers() != 0) {
+				lore.add(ChatColor.DARK_AQUA + "Max Soil Layers: " + ChatColor.GRAY + Integer.toString(plant.getMaximumSoilLayers()));
+			}
+			if (plant.getMaximumSoilBonus() != 0) {
+				lore.add(ChatColor.DARK_AQUA + "Max Soil Bonus: " + ChatColor.GRAY + Double.toString(plant.getMaximumSoilBonus()));
+			}
+			for (Map.Entry<Material, Double> entry : plant.getGreenHouseRates().entrySet()) {
+				lore.add(ChatColor.DARK_AQUA + "Green House Rate: " +  ChatColor.GRAY  + ItemNames.getItemName(entry.getKey())
+						+ " (" + Double.toString(entry.getValue()) + ")");
```

Comment:
Double.toString shouldn't be used in chat outputs, because it's likely to produce something like `3.000000001` at some point. You likely want to wrap it using https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html instead

Reply:
Also if you are inserting arguments like that (or pretty much doing anything other than combining two strings), use String.format(). It's more readable and faster (applies to most of the lore you are adding)

In this case for example something like:
`String.format("%sGreen House Rate: %s%s (%s)", ChatColor.DARK_AQUA, ChatColor.Gray, ItemNames.getItemName(entry.getKey()), decimalFormatter.format(entry.getValue()))`